### ISSUE TITLE
Possible Scaling Issue with the Periodic Table Window

### ISSUE THREAD

**Issue by SeanRyland** [brettbode/wxmacmolplt#115]
### Please describe the problem or enhancement:
I have a second-year undergrad student trying to learn molecular modelling using wxMacMolPLT, and she has reported that the Periodic Table buttons are overlapping and impossible to select the correct element.

### If this is a bug report please list the steps required to reproduce the problem:
She is using a Microsoft Surface (unsure of the model#) running 64-bit Win10 with the stock keyboard/mouse configuration. Opening the "Show Build Tools" option causes the error.  I'm working on getting a screenshot of the error, but can't replicate the issue myself.  Screenshots available below.
![macmol 1](https://user-images.githubusercontent.com/63681554/79381738-4cf01800-7f17-11ea-9b2c-a62c1aa1dba8.png)
![macmol comp specs](https://user-images.githubusercontent.com/63681554/79381740-4d88ae80-7f17-11ea-82ee-d0bdd8228ee0.png)

It doesn't look too bad in the screenshot, but my student reports that she's unable to click on the correct elements, so she's resorted to adding them by hand in the Z-matrix editor, which works, but is cumbersome.

### What version are you running?
wxMacMolPLT 7.7 (64-bit)

### What OS are you using? (please include 32/64 bit and if Linux the specific distribution)
64-bit Win10

**Comment by brettbode** [comment]
Thanks for the report. I have not heard of that issue. Let me know if you can narrow it down a bit.  One thing to note is that when in edit mode with the molecule or periodic table window in front (focus) you can also type the atomic symbol for the desired element to change the selected element. That might provide a workaround in the event you can't properly hit the buttons.

**Comment by SeanRyland** [comment]
Perfect, thanks for the workaround.  Unfortunately, with the self-isolation issue at the moment, I haven't been able to get all the details on her computer or a screenshot of the error, but I'll update ASAP.

**Comment by brettbode** [comment]
The screenshots look like a font sizing issue. It seems strange that it only affects that dialog and not Windows systems in general, which may also make fixing it a bit of a challenge. I'll take a look when I get a few other issues fixed. Thanks for the report.

**Comment by brettbode** [comment]
Hello Sean,
   I have posted a new release (v7.7.2) that appears to work correctly under Windows when the display resolution is higher than the default. Basically the short term solution was to update to the very latest version of wxWidgets and the scaling issue went away. While researching the issue I did learn there are some things I can do better with high-res displays, but at least the just released version should work reasonably well.
